Make some paths opensync use more general so that opensync can find plugins
and formats installed by other packages

--- libopensync-0.22/configure	2008-02-18 16:49:13.000000000 +0000
+++ libopensync-0.22/configure.new	2008-02-18 16:52:05.000000000 +0000
@@ -20882,10 +20882,10 @@
 OPENSYNC_CONFIGDIR=${datadir}/opensync/defaults
 
 
-OPENSYNC_PLUGINDIR=${libdir}/opensync/plugins
+OPENSYNC_PLUGINDIR=@%Compile_goboLibraries%@/opensync/plugins
 
 
-OPENSYNC_FORMATSDIR=${libdir}/opensync/formats
+OPENSYNC_FORMATSDIR=@%Compile_goboLibraries%@/opensync/formats
 
 
 OPENSYNC_HEADERDIR=${includedir}/opensync-1.0/opensync
